src/app/main.cpp

```diff
@@ -134,6 +134,10 @@ int main(int argc, char *argv[])
     bool isOneArg = (argc == 2);
 
     try {
+#if !defined(DISABLE_GUI) && (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
Maybe move this block of code above of the try-catch.

Reply:
firstly I did it :) but later decided to move inside :)
now it is outside try/catch block

Reply:
Sorry, why it's outside Application constructor? I would move it there. Or am I missing something?

Reply:
unfortunately, this flag must be set before QApllication creation... it even produces the output and nothing changes when it is set after QApplication construction (in Application class constructor, for example) see Qt documentation for details...

Reply:
see this comment https://github.com/qbittorrent/qBittorrent/pull/12102#discussion_r387272139 for exact output which is produced when flag was set in constructor

Reply:
>unfortunately, this flag must be set before QApllication creation...

Well, then I suppose you need to place a corresponding comment near it to prevent it from being moved during "yet another" code cleanup.

Reply:
done, comment is added. comment text was copied from app output mentioned above